native_client/CODINGSTYLE.md

```diff
@@ -25,4 +25,4 @@ File naming
 Doubts
 ======
 
-If in doubt, please ask on our IRC channel, `#machinelearning` on `irc.mozilla.org`.
+If in doubt, please ask on our Matrix chat channel: https://matrix.to/#/!VkKaHlfgpCUnNSLuWd:mozilla.org?via=mozilla.org
```

Comment:
Just checked this link and I think it doesn't work. But this link
```
https://chat.mozilla.org/#/room/!USFCphMyGNrPSFlRph:mozilla.org
```
seems to work.